lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -50,6 +50,9 @@ has tag_for => sub {
 
 has_ro ua => sub { Mojo::UserAgent->new->max_redirects(3) };
 
+has [qw(config)];
+my $plugin;
```

Comment:
This won't work. No way I'm accepting globals in my code.

Also, the "config" attribute is not document, nor is it required to make this work.